infrastructure_source_code.md

```diff
@@ -0,0 +1,44 @@
+# DataWorks Open Source Infrastructure Source Code Policy
+
+## Purpose
+
+This document sets out local policy, for the Department for Work and Pensions (DWP) DataWorks team, on what types of information can and cannot be included in infrastructure code that is made publicly available.
+
+It is expected that, at some point, this document will be superceded by a DWP-wide open-source infrastructure code policy which encompasses all DWP public GitHub repositories. Some useful [DWP guidance on creating open source products](https://confluence.service.dwpcloud.uk/pages/viewpage.action?title=Guidance+on+creating+open+source+products&spaceKey=EN) currently exists, but it has not yet been made open-source.
+
+## Scope
+
+This policy covers infrastructure code. Please refer to the [index](README.md#index) for links to other related polices.
+
+## General Approach
+
+Infrastructure code should be made open wherever it is appropriate to do so:
+
+This approach makes code more readily accessible to a broad target audience, so that they can be more easily used for their intended purpose;
+1. This approach enables and encourages re-use of common standards across the DWP and other Government departments, which in turn makes collaboration easier. It also enables other parties to feed back improvements, which benefits everyone.
```

Comment:
For infrastructure code particularly, one would hope that this would be re-used by other parties beyond government if we're the first to do something useful.